test/controllers/api/stories_controller_test.rb

```diff
@@ -16,9 +16,40 @@
 
     it 'can create a new story' do
       story_hash = { title: 'create story', set_account_uri: "/api/v1/accounts/#{account.id}" }
+      @request.env["CONTENT_TYPE"] = "application/json"
       post :create, story_hash.to_json, api_version: 'v1', format: 'json'
       assert_response :success
     end
+
+    it 'can create a new story with url parameters' do
+      @request.env["CONTENT_TYPE"] = "application/json"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.